### PR TITLE
[INLONG-3450][SDK] Sort-SDK change ack log from info to debug

### LINKED ISSUE #3450: [Feature][SDK] Sort-SDK change ack log from info to debug
### Description

Sort-SDK change ack log from info to debug

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by wardlican**
[INLONG-3450]Sort-SDK upgrade pb version to 3.x

### Title Name: [INLONG-XYZ][component] Title of the pull request

where *XYZ* should be replaced by the actual issue number.

Fixes #3450

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve.*

### Modifications

*Describe the modifications you've done.*

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

**Review by healchow — APPROVED** [review]

**Review by vernedeng — APPROVED** [review]

**Review by luchunliang — APPROVED** [review]
ok

File: inlong-sdk/sort-sdk/pom.xml [hunk 1]
```diff
@@ -111,6 +111,18 @@
             <groupId>org.apache.inlong</groupId>
             <artifactId>sdk-common</artifactId>
             <version>${project.version}</version>
+            <exclusions>
```
Comment (dockerzhang) [inline inlong-sdk/sort-sdk/pom.xml:114]: it's better to remove the PB dependency in `sdk-common`?

File: inlong-sdk/sort-sdk/pom.xml [hunk 2]
```diff
@@ -111,6 +111,18 @@
...
+                    <artifactId>protobuf-java</artifactId>
+                    <groupId>com.google.protobuf</groupId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+
+        <dependency>
+            <groupId>com.google.protobuf</groupId>
+            <artifactId>protobuf-java</artifactId>
+            <version>3.19.4</version>
```
Comment (dockerzhang) [inline inlong-sdk/sort-sdk/pom.xml:125]: move the definition of version in the parent property

Comment (wardlican) [reply inlong-sdk/sort-sdk/pom.xml:125]: OK

File: inlong-sdk/sort-sdk/pom.xml (hunk 1 above)
Comment (wardlican) [reply inlong-sdk/sort-sdk/pom.xml:114]: yes

**Review by EMsnap — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**